misc/language-tour-guide/src/AppRoutes.tsx

```diff
@@ -1,10 +1,10 @@
 import { Route, Routes } from "react-router";
 import { Layout } from "./components/Layout";
 import { ExampleLayout } from "./components/ExampleLayout";
-import { createRouteComponents } from "./utils/content/content-routes";
 import { useMemo } from "react";
 import { useScrollToHash } from "./utils/hooks/useScrollToHash";
 import { NotFound } from "./pages/NotFound";
+import { createRouteComponents } from "./utils/content/route-builder";
```

Comment:
Can you please also modify the Routes to show something different than 404 on `/` path?